google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddHotelCallout.java

```diff
@@ -0,0 +1,231 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.hotelads;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v2.common.HotelCalloutFeedItem;
+import com.google.ads.googleads.v2.enums.ExtensionTypeEnum.ExtensionType;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.CampaignExtensionSetting;
+import com.google.ads.googleads.v2.resources.ExtensionFeedItem;
+import com.google.ads.googleads.v2.services.CampaignExtensionSettingOperation;
+import com.google.ads.googleads.v2.services.CampaignExtensionSettingServiceClient;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemOperation;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemServiceClient;
+import com.google.ads.googleads.v2.services.MutateCampaignExtensionSettingsResponse;
+import com.google.ads.googleads.v2.services.MutateExtensionFeedItemsResponse;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.StringValue;
+
+/**
+ * Adds a hotel callout to a specific campaign
+ */
+
+ public class AddHotelCallout {
+
+    private static class AddHotelCalloutParams extends CodeSampleParams {
+
+        public static final String CALLOUT_TEXT = "--calloutText";
```

Comment:
Please add these to ArgumentNames.java

Reply:
Why is this resolved? Looks like it's not done yet.